onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -58,8 +57,26 @@ def fuse_xformers(model: ir.Model) -> None:
     # Finally: inline any intermediate fusion functions introduced that were not
     # consumed by other fusions, and eliminate any remaining unused nodes.
     optimize(model)
+    return model
+
+
+def optimize_for_ort(model: ir.Model, config_name: str | None = None) -> ir.Model:
```

Comment:
Should we call it target? Or maybe config_name is more general?

Reply:
Current usage (in ort/olive) is EP (which is ORT-equivalent to target, I guess). But I think we could envision other configs (at least for debugging, testing, perhaps more generally too). Hence, I made it config_name ...